configure.ac

```diff
@@ -1880,6 +1880,48 @@ then
 fi
 
 
+# Maxim Integrated MAXQ10XX
+ENABLED_MAXQ10XX="no"
+maxqpartnumber=""
+AC_ARG_WITH([maxq10xx],
+    [AS_HELP_STRING([--with-maxq10xx=PART],[MAXQ10XX PART Number])],
+    [
+        AC_MSG_CHECKING([for maxq10xx])
+
+        # Read the part number
+        maxqpartnumber=$withval
+
+        if test "$maxqpartnumber" = "MAXQ1065"; then
+            LIB_STATIC_ADD="$LIB_STATIC_ADD lib/libmaxq1065_api.a"
+            AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_MAXQ1065"
+            ENABLED_MAXQ10XX="yes"
+        elif test "$maxqpartnumber" = "MAXQ108x"; then
+            LIB_STATIC_ADD="$LIB_STATIC_ADD lib/libmaxq108x_api.a"
+            AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_MAXQ108X"
+            ENABLED_MAXQ10XX="yes"
+        else
+            AC_MSG_ERROR([need a valid MAXQ part number])
+        fi
+
+        case $host_os in
+        mingw* | cygwin* | msys* | pw32* | os2* | cegcc*)
+                ENABLED_MAXQ10XX="yes"
+                LIB_ADD="$LIB_ADD -lcrypt32"
```

Comment:
Is this because of `WOLFSSL_SYS_CA_CERTS`? Best option is `--disable-sys-ca-certs`.

Reply:
David, you have added lcrypt32 during compile time via #pragma on Windows. However, this is only compatible with MSVC. This breaks compliance with gcc and msys2.

ca-certs might not be needed for our test purposes. However, this feature is not supported at all for msys like environments in this case.

I needed to update ssl.c to remove dependency on pragma for library addition as well.

![image](https://user-images.githubusercontent.com/46590259/202386207-2f2e48a4-688b-442d-a30f-8908c3c8c60a.png)

As I flagged earlier, this dependency should be move under corresponding switch/option in wolfssl. If that is ca-certs that works.

You need to add following line:

case $host_os in
mingw* | cygwin* | msys* | pw32* | os2* | cegcc*)
	LIB_ADD="$LIB_ADD -lcrypt32"
	;;
esac

Reply:
@mehmetariman 

I think what David is trying to say is if you use `--disable-sys-ca-certs` then that pragma is no longer in effect.  This configure time flag is appropriate for MAXQ as it will not use platform supplied CA certs; it uses CA Certs that are stored internally.